src/gallium/state_trackers/nine/iunknown.c

```diff
@@ -177,6 +177,7 @@ NineUnknown_SetPrivateData( struct NineUnknown *This,
     }
 
     header->size = SizeOfData;
+    header->data = (void *)header + sizeof(*header);
```

Comment:
Thinking about it, it's not neccessary to store the pointer in every header, it can be calculated. I'll send a fixed version